plutus-core/plutus-ir/test/PlutusIR/Transform/Inline/callsite-6970

```diff
@@ -0,0 +1,51 @@
+{-
+let
+  !f : all a. list integer -> list integer
+    = /\a -> \(xs : list integer) -> tailList {integer} xs
+in
+\(xs : list integer) ->
+  headList
+    {integer}
+    (tailList {integer} (f {integer} (f {integer} xs)))
+-}
+
+(let
```

Comment:
If only this file had an extension `.uplc` then syntax highlighting would have worked out of the box.

Reply:
This is PIR though.

Reply:
I am confused: this file contains both and IIUC PIR is inside a comment (I don't expect code in the comment to be syntax-highlighted) but UPLC is outside (I do expect it to be syntax-highlighted). 

Why don't you go with the regular approach: one file for PIR and a separate one for UPLC ?

Anyhow, I find a good practice to use file extensions that hint at a media-type.